app/transport/http/server.go

```diff
@@ -32,3 +37,7 @@ func (srv *Server) Run() error {
 
 	return nil
 }
+
+func attachUserEndpoints(router *mux.Router, handler UserHandler) {
```

Comment:
The function with `handler UserHandler` is good enough for now, but it is not suitable if we need to create other endpoint  of different entity, for example "Tasks". Do we change it in next iteration of the development?